### PR TITLE
Deactivate saving of reads and other large files for full tests

### PR DESCRIPTION
**Pull request by jfy133**
Re: https://nfcore.slack.com/archives/CTY8L26C8/p1679040181299829?thread_ts=1678923529.023299&cid=CTY8L26C8 and https://nfcore.slack.com/archives/DEL52B0U8/p1679040013345729

Hopefully if I can get it to work, in teh future we can have a system to save the final reads that go into preprocessing a la the diea at the bottom fo #262, and save only those

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_full.config [hunk 1]
```diff
@@ -19,52 +19,52 @@ params {
...
-    run_motus = true
-    motus_save_mgc_read_counts = true
-
-    run_profile_standardisation = true
-    run_krona = true
+    save_preprocessed_reads             = false
+
+    perform_shortread_qc                = true
+    shortread_qc_mergepairs             = true
+    perform_shortread_complexityfilter  = false
```
Comment (sofstam) [inline conf/test_full.config:26]: Shall we also  deactivate this flag?

Comment (jfy133) [reply conf/test_full.config:26]: Oh oops! I got overzealosu

Comment (jfy133) [inline conf/test_full.config:26]:
```suggestion
    perform_shortread_complexityfilter  = true
```

File: conf/test_full.config [hunk 2]
```diff
@@ -19,52 +19,52 @@ params {
...
+    kraken2_save_reads                  = false
+    kraken2_save_readclassification     = false
+    kraken2_save_minimizers             = false
+
+    run_krakenuniq                      = true
+    krakenuniq_save_reads               = false
+    krakenuniq_save_readclassifications = false
+
+    run_bracken                         = true
+
```
Comment (jfy133) [inline conf/test_full.config:56]:
```suggestion
```

File: conf/test_full.config [hunk 3]
```diff
@@ -19,52 +19,52 @@ params {
...
+        
+    run_malt                            = true
+    malt_save_reads                     = false
+    malt_generate_megansummary          = true
+        
+    run_metaphlan3                      = true
+        
+    run_motus                           = true
+    motus_save_mgc_read_counts          = true
+
```
Comment (jfy133) [inline conf/test_full.config:65]:
```suggestion

    run_metaphlan3                      = true

    run_motus                           = true
    motus_save_mgc_read_counts          = true

```

**Review by sofstam — APPROVED** [review]

**Review by Midnighter — APPROVED** [review]

**Pull request merged**